elrond-wasm-debug/src/blockchain_mock.rs

```diff
@@ -83,6 +83,7 @@ pub struct BlockInfo {
 	pub block_nonce: u64,
 	pub block_round: u64,
 	pub block_epoch: u64,
+	pub block_random_seed: Box<[u8;48]>,
```

Comment:
I don't see it being used. Add get_block_random_seed in ext_mock.rs.

You also need to test it in a contract.
Add a function "get_block_random_seed_endpoint" in basic-features and write a mandos test with it.